source/constructs/api/common/db_base_col.py

```diff
@@ -1,26 +1,30 @@
 import os
 from datetime import datetime
 from .constant import const
-import re
 
 
 def before_exec(conn, clause, multi_params, params):
-    if multi_params:
-        # The public fields for batch execution are processed by the business module itself
-        pass
-    else:
-        __handle_params(clause, params)
-
-
-def __handle_params(clause, params):
+    operator = os.getenv(const.USER, const.USER_DEFAULT_NAME)
+    operation_time = datetime.utcnow()
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Naive datetime objects are treated by many datetime methods as local times, We recommend that you create aware timestamps with the `now` or `fromtimestamp` functions using the `tzinfo` to attribute to represent times in UTC. Do not use the [datetime.utcnow()](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) because it returns only a `naive` datetime object. 

[Learn more](https://docs.python.org/3/library/datetime.html#aware-and-naive-objects)